test/unit/StakingProducts/setProducts.js

```diff
@@ -1,6 +1,7 @@
 const { expect } = require('chai');
```

Comment:
we should have a test here that best captures the error in prod, where the actual price was sitting at the initial targer price until the manager updated the target which caused it to drop and assert around that

Reply:
Test added